e2e/ctl_v3_test.go

```diff
@@ -115,6 +124,10 @@ func testCtl(t *testing.T, testFunc func(ctlCtx), opts ...ctlOption) {
 	}
 	ret.applyOpts(opts)
 
+	if ret.flagByEnv {
```

Comment:
@gyuho you can do

useEnv := ret.envMap == nil

then we can use useEnv as flagByEnv.